logs: added informative logs

RanRegev opened the pull request:
during the work on 4.7.9 certification added
logs that may serve other users as well

Signed-off-by: Ran regev <ran.regev@devalore.com>

tomereli commented on common/beerocks/bcl/source/beerocks_os_utils.cpp:
```diff
@@ -81,7 +81,7 @@ std::string os_utils::system_call(std::string cmd, int log_lvl, bool detached)
 
     if (system(cmd.c_str()) != 0) {
-        LOG(ERROR) << "System returned non-zero";
+        LOG(ERROR) << "System returned non-zero for the command:" << cmd.c_str();
```
If you're already changing this, why not also include the return code?
```
int ret = system(cmd.c_str());
LOG_IF(!ret, ERROR) << cmd << " failed with return code " << ret;
```

tomereli commented on framework/transport/ieee1905_transport/ieee1905_transport_local_bus.cpp:
```diff
@@ -83,7 +83,9 @@ void Ieee1905Transport::handle_local_bus_cmdu_tx_message(CmduTxMessage &msg)
         }
 
         if (msg.metadata()->msg_type != ntohs(ch->messageType)) {
-            MAPF_WARN("CmduTxMessage messageType mismatch.");
```
OK, but this should be a separate commit.

tomereli reviewed (approved):
The change itself is good, but the split to commit is not. Each change should be a separate commit, please split accordingly before merging.

vitalybu commented on common/beerocks/bcl/source/beerocks_os_utils.cpp:
```diff
@@ -81,7 +81,7 @@ std::string os_utils::system_call(std::string cmd, int log_lvl, bool detached)
+        LOG(ERROR) << "System returned non-zero for the command:" << cmd.c_str();
     }
 
 //if (log_lvl == 2) LOG(INFO) << "system_call ret:\n" << ret_str << "\n";
```
Also a good oportunity to remove this comment

vitalybu reviewed (approved):
@tomereli has some valid comments, but since they are minor, approving anyway.
@ranregev please implement the changes before merging.

RanRegev committed:
common: os utils: added informative log

the log of failed system command now print:
* the failed command
* the return value of the failed command

Signed-off-by: Ran regev <ran.regev@devalore.com>

RanRegev committed:
framework: ieee1905_transport: informative log message

for debugging:
print details about mismatch between the packet's
message-type and the message's message-types

Signed-off-by: Ran regev <ran.regev@devalore.com>

RanRegev committed:
common: ucc listener: print the value of the onvalid value

Added a print to print the value that was found to be invalid

Signed-off-by: Ran regev <ran.regev@devalore.com>

RanRegev committed:
cppcheck: resolved unintentialy a cppcheck error

Signed-off-by: Ran Regev <ran.regev@devalore.com>

RanRegev committed:
cppcheck: added override specifier as indicated

Signed-off-by: Ran Regev <ran.regev@devalore.com>

RanRegev committed:
cppcheck-diff: replaced new file with theold one

Signed-off-by: Ran Regev <ran.regev@devalore.com>
